node/src/reactor/main_reactor.rs

```diff
@@ -1168,6 +1169,7 @@ impl reactor::Reactor for MainReactor {
             max_attempts: config.node.max_attempts,
             idle_tolerance: config.node.idle_tolerance,
             control_logic_default_delay: config.node.control_logic_default_delay,
+            shutdown_for_upgrade_timeout: config.gossip.validate_and_store_timeout * 2,
```

Comment:
I'm not opposed to it, but I'm curious about the reasoning behind this choice.

Reply:
It stems from yesterday's discussion at stand-up. Basically the thing that can hold us up is gossiping signatures, so we talked about making the timeout here related to some kind of gossip timeout, and if I understood correctly, this value in particular was pointed out as one that corresponds to what we want.

Reply:
Could we maybe have this option configurable through the node config and maybe default to this value?

Reply:
That sounds like a good idea :+1: